stubs/core.stub

```diff
@@ -315,16 +321,16 @@ function is_callable(mixed $value, bool $syntax_only = false, ?string &$callable
 function abs($num) {}
 
 /**
- * @return ($string is lowercase-string ? lowercase-string : string)
+ * @return ($string is lowercase-string ? lowercase-string : ($string is uppercase-string ? uppercase-string : string))
```

Comment:
Should check `$string is lowercase-string&uppercase-string` first.
https://phpstan.org/r/98dd06f1-2543-45c5-96b9-58c0495e3302

or we will have the following issue:
```
'-23-' // isLowercase TRUE ; isUppercase TRUE
trim('-23-', '-') // isLowercase TRUE ; isUppercase FALSE
```

Reply:
Resolved in 2f78f966a